server/src/main/java/io/spine/server/integration/ConfigurationChangeObserver.java

```diff
@@ -89,7 +89,7 @@ public void handle(ExternalMessage value) {
         clearStaleSubscriptions(request.getRequestedMessageTypeList(), origin);
         if (!knownContexts.contains(origin)) {
             knownContexts.add(origin);
-            integrationBus.notifyOfCurrentNeeds();
+            integrationBus.introduceSelf();
```

Comment:
This doesn't sound right. The bus is already introduced to the system. It's the new context which introduces itself. Or, if you want the bus to do the job the method should be named differently.